tests/utils/leco/test_utils.py

```diff
@@ -19,3 +19,39 @@ def test_data_actuator():
     value = DataActuator(data=10.5)
     serialized = serialize_object(value)
     assert isinstance(serialized, str)
+    # with Serializer
+    # expected = "AAAADERhdGFBY3R1YXRvcgAAAAM8ZjgAAAAIc8Pcmx7U2UEAAAAIYWN0dWF0b3IAAAADcmF3AAAABkRhdGEwRAAAAAd1bmlmb3JtAAAAAQAAAAVhcnJheQAAAAM8ZjgAAAAIAAAAAQAAAAEAAAAAAAAlQAAAAAAAAAABAAAABnN0cmluZwAAAARDSDAwAAAAAAAAAAAAAAAAAAAAAAAAAAA="
+    # with Serializer Factory
+    expected = "AAAADERhdGFBY3R1YXRvcgAAAAVmbG9hdAAAAAM8ZjgAAAAIYGa0OSDU2UEAAAADc3RyAAAACGFjdHVhdG9yAAAAA3N0cgAAAANyYXcAAAADc3RyAAAABkRhdGEwRAAAAANzdHIAAAAHdW5pZm9ybQAAAARsaXN0AAAAAQAAAAduZGFycmF5AAAAAzxmOAAAAAgAAAABAAAAAQAAAAAAACVAAAAAA3N0cgAAAAAAAAAEbGlzdAAAAAEAAAADc3RyAAAABENIMDAAAAADc3RyAAAAAAAAAARsaXN0AAAAAAAAAARsaXN0AAAAAAAAAARsaXN0AAAAAAAAAARsaXN0AAAAAA=="
```

Comment:
beware that between windows and linux some encoding of integers to bytes are different (number of bytes to encode it as default). The direct testing like this could fail in one or the other

Reply:
Ok.
So, we have to do some specific tests (I suggest that it works on the github runners).